850.split-ambiguities/e.yaml

```diff
@@ -97,6 +97,12 @@
 
 - { name: enigma, wwwpart: sgtatham, setname: enigma-sgtatham }
 
+- { name: envision, wwwpart: gabmus, setname: envision-xr }
```

Comment:
Please also add `envision-xr` -> `envision` rule in 800/x ruleset, so all packages come through this rule, and are affected if it's changed.